src/components/ReportActionItem/TaskView.js

```diff
@@ -102,7 +101,7 @@ function TaskView(props) {
                                             containerBorderRadius={8}
                                             caretSize={16}
                                             accessibilityLabel={taskTitle || props.translate('task.task')}
-                                            disabled={isCanceled || !canModifyTask}
+                                            disabled={disableState}
```

Comment:
I think it's more readable directly using `Task.canModifyTask(props.report, props.currentUserPersonalDetails.accountID)`. Similar to what we did for other components.